content/docs/command-reference/repro.md

```diff
@@ -97,17 +99,35 @@ up-to-date and only execute the final stage.
 
 ## Options
 
-- `-f`, `--force` - reproduce a pipeline, regenerating its results, even if no
-  changes were found. This executes all of the stages by default, but it can be
-  limited with the `targets` argument, or the `-s`, `-p` options.
+- `targets` (optional command argument) - what to reproduce (the pipeline(s) in
+  `./dvc.yaml` by default). Different things can be provided as targets
+  depending on the flags used (more details in each option). Examples:
+
+  - `dvc repro linear/dvc.yaml`: Specific `dvc.yaml` file to reproduce
+  - `dvc repro -R pipelines/`: Directory path to explore recursively for for
```

Comment:
```suggestion
  - `dvc repro -R pipelines/`: Directory path to explore recursively for
```

Reply:
Oops thanks! Want to send a quick typo fix PR since this is merged?